rpc/documentation/api.md

```diff
@@ -627,6 +627,23 @@ but not all sidechain blocks may be known by the wallet.
 
 ___
 
+#### `GetAccountExtendedPubKey`
+
+The `GetAccountExtendedPubKey` method queries the wallet for an account pubkey. If the provided account number is beyond the current account index, then it returns an error.
```

Comment:
keep the documentation wrapped to 80 columns please

We also don't need to document in the description that a nonexistent account will cause an error, that's expected (and already documented in the expected errors section below).